src/git-version.cr

```diff
@@ -10,13 +10,11 @@ module GitVersion
 
   DEV_BRANCH_SUFFIX = "SNAPSHOT"
 
-  MASTER_BRANCH = "master"
-
   MAJOR_BUMP_COMMENT = "breaking:"
   MINOR_BUMP_COMMENT = "feature:"
 
   class Git
-    def initialize(@dev_branch : String, @folder = FileUtils.pwd)
+    def initialize(@dev_branch : String, @release_branch : String, @folder = FileUtils.pwd)
```

Comment:
I think it's worth to provide the default value

Reply:
Added :)